gusto/forcing.py

```diff
@@ -89,7 +94,10 @@ def forcing_term(self):
         L = self.scaling * L
         # sponge term has a separate scaling factor as it is always implicit
         if self.sponge:
-            L -= self.mu_scaling*self.sponge_term()
+            L -= self.stage*self.state.timestepping.dt*self.sponge_term()
+        # hydrostatic term has no scaling factor
+        if self.hydrostatic:
+            L += self.hydrostatic_term()
```

Comment:
OK, so in the hydrostatic case, in the first step you add `-<u0 . k, test . q>` and then in the next step you add `<u0 . k, test . q>`.

Whereas the sponge term is only added on the second step.

Reply:
Is it clear that you want the implicitness of the sponge term to be identical to the hydrostatic term?

Reply:
The hydrostatic terms here are a result of the missing time derivative in the vertical velocity equation, and so aren't really part of the forcing but rather part of what function the forcing should be added to (that's why they have no scaling factor). I then thought that since an explicit/implicit switch already existed, using ```(2*self.stage-1)``` would be the least intrusive way to implement this.